d8content/metadatadisplays/iiif_presentation_api_3.0_manifest.twig.html

```diff
@@ -1,18 +1,10 @@
 {# This Template generates IIIF 3.0 Presentation API Manifest from a Strawberry Field
 JSON.
-​
-Updated 07/98/2022 for 1.0.0
+
+Updated 01/20/2023 for 1.1.0
 
 Changes:
-  - Better Labeling for Items
-  - Adds Rights
-  - Fixes on Canvas IDs
-  - Deals with PDFs too, joining them all together
-  - Adds MarkDown support for Description
-  - Fixes Structures/Canvas IDs
-  - Adds Annotations for Images
-  - Adds PDF/Document/pages
-  - Adds Thumbnail for Video
+  - Fixes mirador_path to handle both standard and self-closing polygon tags.
```

Comment:
Please keep the previous change list for now! (will make release easier, thanks)

Reply:
@DiegoPino Sorry, mixed up the change list for the 2 templates. Will make another commit shortly.

Reply:
Fixed in 0fa9b379fca00809a69619ca211aa5a5f33b05f8.

Reply:
Thanks!